README.md

```diff
@@ -120,15 +114,15 @@ of `x` within `@byrow!`.
 `byrow!` also supports special syntax for allocating new columns to make
 `byrow!` more useful for data transformations. The syntax `@newcol
 x::Array{Int}` allocates a new column `:x` with an `Array` container with eltype
-`Int`. Note that the returned AbstractDataFrame includes these new columns, but
+`Int`. Note that the returned `AbstractDataFrame` includes these new columns, but
 the original `df` is not affected. Here is an example where two new columns are
 added:
 
 ```julia
 df = DataFrame(A = 1:3, B = [2, 1, 2])
 df2 = @byrow! df begin
     @newcol colX::Array{Float64}
-    @newcol colY::DataArray{Int}
+    @newcol colY::Array{Int}
```

Comment:
Maybe use `Union{Int,Missing}` to illustrate that missing values are supported?

Reply:
fixed and added a case with `missing` to show that it actually works.